credhub.yml

```diff
@@ -0,0 +1,107 @@
+# Update with local file location
+- type: replace
+  path: /releases/-
+  value:
+    name: credhub
+    url: file:///tmp/credhub-0.3.1.tgz
+
+# Update dev_key with chosen value
+- type: replace
+  path: /instance_groups/name=bosh/jobs/-
+  value:
+    name: credhub
+    release: credhub
+    properties:
+      credhub: 
+        port: 8844
+        user_management:
+          uaa:
+            url: "https://((internal_ip)):8443"
+            verification_key: ((uaa_jwt_signing_key.public_key))
+        data_storage:
+          type: postgres
+          host: 127.0.0.1
+          port: 5432
+          username: postgres
+          password: ((postgres_password))
+          database: credhub
+        tls: ((credhub_tls))
+        log_level: info
+        encryption:
+          provider: dev_internal
+          dev_key: ABCDEF0123456789ABCDEF0123456789 #Replace with chosen value - 32 hex characters 
+
+- type: replace
+  path: /instance_groups/name=bosh/properties/postgres/additional_databases?/-
+  value: credhub
+
+# Configure Director
+- type: replace
+  path: /instance_groups/name=bosh/properties/director/config_server?
+  value:
+    enabled: true
+    url: "https://((internal_ip)):8844/api/"
```

Comment:
I think the internal IP is preferred here. The justification [here][1] - in context of why not to issue certificates with a 'localhost' entry - is compelling to me. 

> In empirical testing, we've seen multiple resolvers (notably, OS X's) send
localhost queries to the network, either as part of DNS suffix searches
(localhost.my.suffix.search.example), as a single-label entry
("\09localhost\0" in DNS terms), or as part of multicast DNS.

> As a result, accessing "https://localhost", say, on a hostile WiFi access
point (such as your coffee shops) can be intercepted by a network attacker
and redirected to a site (or a certificate) of their choosing.

[1]:https://cabforum.org/pipermail/public/2015-June/005673.html